src/internal/NeoEsp32I2sXMethod.h

```diff
@@ -333,17 +345,17 @@ class NeoEsp32I2sMuxBus
     private:
         static NeoEspI2sContext s_context0;
     };
-#endif
 
-class NeoEsp32I2s1Mux8Bus : public NeoEsp32I2sMuxBus
-{
-public:
-    NeoEsp32I2s1Mux8Bus() : NeoEsp32I2sMuxBus(1, s_context1)
+    class NeoEsp32I2s1Mux8Bus : public NeoEsp32I2sMuxBus
```

Comment:
did this line get indented for a reason?

Reply:
NeoEsp32I2s1Mux8Bus  is just moved to esp32 ifdef section (won't make sense for esp32-s2 since it doesn't have i2s1)